src/api/app/helpers/webui/staging/workflow_helper.rb

```diff
@@ -23,10 +23,11 @@ def review_progress(staging_project)
   end
 
   def testing_progress(staging_project)
-    notdone = staging_project.checks.pending.size
     allchecks = staging_project.checks.size + staging_project.missing_checks.size
+    return 0 if allchecks == 0
+
+    notdone = staging_project.checks.pending.size
 
-    return 100 if allchecks == 0
     100 - notdone * 100 / allchecks
```

Comment:
shouldn't we add parenthesis here to make sure that we are getting the right results?

Reply:
Adding the parenthesis where exactly? The formula looks good to me as it is.

Reply:
if you want to have ```100 - ((notdone * 100) / allchecks)``` then you are right.. I thought you wanted to have ```((100 - notdone) * 100) / allchecks```